kfdefs/base/jupyterhub/xskipper/jupyterhub-spark3.0.1-singleuser-profiles-configmap.yaml

```diff
@@ -0,0 +1,78 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: jupyter-spark3.0.1-singleuser-profile
```

Comment:
nitpick: I would recommend including `xskipper` in the file name and configMap name so that it clearly identifies as a singleuser profile for notebooks with xskipper